### ISSUE TITLE
Sub-projects support

### ISSUE THREAD

**Issue by realloc** [iRessources/AgileDwarf#11]
Are there any plans on supporting sub-projects? 
I need to see tasks form all sub-projects in top-level project's "Task" and "Sprints" tabs.

**Comment by markablov** [comment]
Do you think that this feature does not lead to some mess?

**Comment by realloc** [comment]
It wouldn't lead to a mess. For one team working on a set of projects
simultaneously you need a common board to see the current state of
development process. Otherwise you will need to switch between boards,
memorize story statuses and assignees and try to imagine the overall
picture.

In the "common board" stories from different sub-projects could be
color-coded to avoid mess.

**Comment by markablov** [comment]
Ok, we'll think about checkbox "include sub-project tasks".

**Comment by realloc** [comment]
That would be great, Thank You!
Currently only the absence of this feature stops me from using AgileDwarf in my team.

**Comment by rdafoe** [comment]
This would be a very useful enhancement.

**Comment by krewetka** [comment]
I agree, this would be great.

Our structure is main project for all "general" issues + few different subproject for frontend, backend, mobile app etc.

Without option to see all issues it is useless for us. There is no way we make the same sprints in 5-10 projects and assign tasks in each one separately.

Without overview there is no easy way to plan following sprint and to follow it later :(

Btw, also "general" overview of all projects would be usefull as people are working on few separate projects ( not subprojects) in the same time and now have to check each project to see their work plan.

**Comment by krewetka** [comment]
"Ok, we'll think about checkbox "include sub-project tasks". - there is already this setting in Redmine in general. Check "administration"->"settings"->"issue tracking". There is "Display subprojects issues on main projects by default" -  you could use this setting :)

**Comment by svolpe** [comment]
I also agree that adding in sub-project support is a must.  My team can not even consider using this tool until that support is available.

**Comment by krewetka** [comment]
Any time span when you plan to implement this? I can quote svolpe: " My team can not even consider using this tool until that support is available". Honestly, if you have structure using subprojects, then right now there is no way to use this tool :/

**Comment by rdafoe** [comment]
markablov said: "Ok, we'll think about checkbox "include sub-project tasks".

Have you decided if you will do this. If so I'll continue to check back, if not perhaps someone else may decide to do. Thanks.

**Comment by torotil** [comment]
I'd like to have sprints follow the same scheme as the version sharing (not shared, sub projects, project hierarchy, all projects).

We are a small team that works on up to three (mostly unrelated) projects at a time. To track our sprints we use a version "shared with all projects" and add all issues for this sprint to the sprint-version.

**Comment by jmbiollaz** [comment]
+1
The possibility to have an on-demand overview of subprojects tasks would be very useful. When you work in a team on several small projects this feature is essential. Without it you constantly have to navigate from one project to the other to compile information.

**Comment by chasingmaxwell** [comment]
Would love to see this feature implemented. Any update?

**Comment by davidrahusen** [comment]
+1
Any news on that topic?

**Comment by folksilva** [comment]
+1
A simple solution updating the models:
https://github.com/folksilva/AgileDwarf/blob/master/app/models/sprints.rb
https://github.com/folksilva/AgileDwarf/blob/master/app/models/sprints_tasks.rb
